app/models/barcode_item.rb

```diff
@@ -28,18 +28,6 @@ class BarcodeItem < ApplicationRecord
 
   alias_attribute :item, :barcodeable
 
-=begin
-  # TODO - BarcodeItems should be able to filter on CanonicalItemId
```

Comment:
I think the `# TODO` needs to stay, but the commented-method is fine to be deleted.